spec/models/bgs_dependents/child_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe BGSDependents::Child do
+  let(:child_info) do
+    {
+      'does_child_live_with_you' => false,
+      'child_address_info' => {
+        'person_child_lives_with' => { 'first' => 'Bill', 'middle' => 'Oliver', 'last' => 'Bradsky' },
+        'address' => {
+          'country_name' => 'USA',
+          'address_line1' => '1100 Robin Cir',
+          'city' => 'Los Angelas',
+          'state_code' => 'CA',
+          'zip_code' => '90210'
+        }
+      },
+      'place_of_birth' => { 'state' => 'CA', 'city' => 'Slawson' },
+      'child_status' => { 'biological' => true },
+      'previously_married' => 'Yes',
+      'previous_marriage_details' => { 'date_marriage_ended' => '2018-03-04', 'reason_marriage_ended' => 'Death' },
+      'full_name' => { 'first' => 'John', 'middle' => 'oliver', 'last' => 'Hamm', 'suffix' => 'Sr.' },
+      'ssn' => '370947142',
+      'birth_date' => '2009-03-03'
+    }
+  end
```

Comment:
Code smell. Use factories here for test setup.

Reply:
I've replaced the fixture with a factory.